cluster-provision/k8s/1.18/provision.sh

```diff
@@ -158,6 +158,11 @@ echo "net.netfilter.nf_conntrack_max=1000000" >> /etc/sysctl.conf
 
 systemctl restart NetworkManager
 
+nmcli connection modify "System eth0" \
+   ipv6.method auto \
+   ipv6.addr-gen-mode eui64
```

Comment:
The method should be `auto` by default, so something is explicitly disabling it. Maybe this comes from the cloud image?

In addition, the addr-gen-mode does not need to be changed, I do not know why this was added.

Reply:
> The method should be `auto` by default, so something is explicitly disabling it. Maybe this comes from the cloud image?
> 

Maybe, RHEL 8 CSB comes with it disabled.

> In addition, the addr-gen-mode does not need to be changed, I do not know why this was added.

I know but i dont see it hurt, it just means that if there are couple nodes it will assign ip according mac.